### PR TITLE
[VAS-687] feat: Add delegation detail page

### PR DESCRIPTION
**Pull request by gioelemella**
#### List of Changes

- added delegation detail list
- added unit test

#### Motivation and Context
[VAS-687](https://pagopa.atlassian.net/browse/VAS-687)

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

[VAS-687]: https://pagopa.atlassian.net/browse/VAS-687?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/pages/delegations/detail/DelegationDetailOperativeTable.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        borderRadius: 2,
+        padding: 3,
+        minHeight: '310px',
+        display: 'flex',
+        flexDirection: 'column',
+      }}
+    >
+      <Typography variant="overline">{t('delegationDetailPage.operativeTable.title')}</Typography>
+
+      <Box key={`operative-table-deatil`} mt={1} data-testid="operative-table-column">
```
Comment (svariant) [inline src/pages/delegations/detail/DelegationDetailOperativeTable.tsx:24]: Useless key

File: src/pages/delegations/detail/DelegationStationDetailsDrawer.tsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+  const history = useHistory();
+
+  return (
+    <PaddedDrawer
+      openDrawer={drawerValue?.station_code !== undefined}
+      setOpenDrawer={setDrawerValue}
+    >
+      <TitleBox title={t('delegationDetailPage.stationDetail.title')} variantTitle="h5" />
+      <Box
+        key={`station-detail-${drawerValue.station_code}`}
```
Comment (svariant) [inline src/pages/delegations/detail/DelegationStationDetailsDrawer.tsx:33]: Useless key

**Review by svariant — APPROVED** [review]
🚀🌕

**Review by jacopocarlini — APPROVED** [review]
👍

**Pull request merged**